src/components/shared/ConstructionPage.tsx

```
@@ -0,0 +1,107 @@
+        </p>
+
+        {/* Back Button */}
+        <button
+          onClick={() => window.history.back()}
+          className={`inline-flex items-center px-6 py-3 rounded-lg font-semibold transition-all duration-200 ${
+            theme === 'dark'
+              ? 'bg-orange-500 hover:bg-orange-600 text-white'
+              : 'bg-orange-500 hover:bg-orange-600 text-white'
+          } transform hover:scale-105 shadow-lg hover:shadow-xl`}
+        >
+          <svg 
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)